pydoctor/test/test_epydoc2stan.py

```diff
@@ -239,8 +282,8 @@ def f(a, b):
     classic_fmt = docstring2html(classic_mod.contents['f'])
     assert annotation_fmt == classic_fmt
 
-@pytest.mark.xfail
 def test_func_arg_when_doc_missing() -> None:
+    # Output with annotated args left undocumented are the same as undocumented
     annotation_mod = fromText('''
     def f(a: List[str], b: int) -> bool:
         """
```

Comment:
These is something that bothers me, which is probably the reason why this test has been marked as xfail.
We are actually loosing some data in the second test down here at line 294. The `@type:` fields are getting ignored if they are not accompanied by a `@param:` field. They should be included included in the documentation for sure, so maybe we should consider

Reply:
> They should be included included in the documentation for sure.

Hrmm, why? Has this been the case always? One might expect missing param docs to be treated the same way regardless of how their types are defined, right?